openedx/openedx-platform
# MICROBA-905 Add django-simple-history  to CertificateInvalidation model

## Description

Adds the `certificates_historicalcertificateinvalidation` table, which will automatically track changes to the `certificates_certificateinvalidation` table using **[django-simple-history](https://django-simple-history.readthedocs.io/en/latest/index.html)**.

This will help **developers** debug changes to course certificates.

## Supporting information

In fulfillment of [MICROBA-905](https://openedx.atlassian.net/browse/MICROBA-905).

## Testing instructions

In the **Instructor Dashboard** for a course, in the **Certificates** tab, see the **Invalidate Certificates** tool. Adding or removing an entry here should add a new entry to our new `certificates_historicalcertificateinvalidation` table.

## Deadline

None (but sooner is definitely better)

## Other information

This requires a migration!
